### ISSUE TITLE
Release for 2.13.0-M1

### ISSUE THREAD

**Issue by ghost** [typelevel/catalysts#14]
and bump the insanely low version number at the same time.

Needed for https://github.com/typelevel/cats/issues/1648

**Comment by SethTisue** [comment]
@BennyHill is this something you might be able to give some attention to sometime soon...? or something you could use external help with?

**Comment by ghost** [comment]
Fyi, this blocked by [scoverage](https://github.com/scoverage/scalac-scoverage-plugin/issues/205) that is in turn blocked by [scala-logging](https://github.com/typesafehub/scala-logging/pull/94)

Also see [comments is cats](https://github.com/typelevel/cats/issues/1648#issuecomment-324439887)

The general "problem" here is that catalysts is the first in the lower levels to pull in more of the regular QA/util libs and so is the first to be "hit" if there are issues. Just saying as to why it can appear to be a bottleneck.

**Comment by ghost** [comment]
@SethTisue scala-logging is in the Lightbend stable, is this something you could look at?

**Comment by ghost** [comment]
Also waiting for https://github.com/tpolecat/tut/issues/183 and after that, https://github.com/47deg/sbt-microsites/pull/215 needs bumping before release

**Comment by SethTisue** [comment]
> @SethTisue scala-logging is in the Lightbend stable, is this something you could look at?

sort of in the Lightbend stable... it was Heiko's project, but he left the company a few years ago and the library has been community-maintained since then. we're more conservative these days about adding new libraries under the typesafehub or lightbend orgs.

but anyway, sorry for the delayed reply, I've commented on https://github.com/typesafehub/scala-logging/issues/93

**Comment by SethTisue** [comment]
scala-logging is now published for 2.13.0-M1, and I submitted a new PR bumping it to M2 (typesafehub/scala-logging#97)

**Comment by SethTisue** [comment]
scala-logging 3.7.2 is now published for M2 as well

**Comment by SethTisue** [comment]
Scala 2.13.0-M3 is now out

**Comment by ghost** [comment]
Ref https://github.com/BennyHill/catalysts/tree/fix/14 - bulk of work done already 😄

**Comment by LukaJCB** [comment]
@BennyHill Any update? :)

**Comment by ghost** [comment]
Sorry, mid-term, 2 week school holidays... looking now :wink:

**Comment by ghost** [comment]
M3 released after https://github.com/typelevel/catalysts/pull/16

**Comment by SethTisue** [comment]
if I add this to a test project:

```diff
+    "org.typelevel"                    %% "catalysts"                 % "0.6",
```

a number of catalysts artifacts do resolve, but then I get:

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.typelevel#catalysts-tests_2.13.0-M3;0.6: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

**Comment by ghost** [comment]
The tests are just internal tests, not published. So this looks correct

**Comment by ghost** [comment]
Oh... We should not be publishing a catalysts artifact in fact. That's the issue. So we can fix that. But just add the libs needed. Eg catalysts-macros, catalysts-platform

**Comment by ghost** [comment]
See #19

**Comment by SethTisue** [comment]
scala/make-release-notes@b9fb9a85c084460180603bc2e67dc9029245a43d
